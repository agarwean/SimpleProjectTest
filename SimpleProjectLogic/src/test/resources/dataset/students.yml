student:
    - student_id: 1
      address: ul. Pierwsza
      birth_date: 1980-12-02
      birth_place: Szczecin
      email: malpa@gmail.com
      forename: Krzysztof
      name: Jarzyna
      pesel: 80120223987
      phone_number: 677788344
      state: slaskie