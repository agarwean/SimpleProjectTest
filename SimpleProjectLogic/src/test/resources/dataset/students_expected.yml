student:
    - student_id: 1
      address: ul. Po zmianie
      birth_date: 1981-12-02
      birth_place: Szczecin po zmianie
      email: malpa_po_zmianie@gmail.com
      forename: Krzysztof po zmianie
      name: Jarzyna po zmianie
      pesel: 123456789012
      phone_number: 12345678
      state: malopolskie